var/spack/repos/builtin/packages/py-stratify/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyStratify(PythonPackage):
+    """Vectorized interpolators that are especially useful for
+    Nd vertical interpolation/stratification of atmospheric and
+    oceanographic datasets.
+    """
+
+    homepage = "https://github.com/SciTools-incubator/python-stratify"
+    url      = "https://github.com/SciTools-incubator/python-stratify/archive/v0.1.tar.gz"
```

Comment:
Can you use the pypi.io download link (see eg `py-numpy` for how it is constructed)

Reply:
Sure - I didn't realise the package was available there.

Reply:
Hi @healther , I have switched the download to pypi.io in 56d282f .